spring-cloud-app-broker-autoconfigure/src/main/java/org/springframework/cloud/appbroker/autoconfigure/AppBrokerAutoConfiguration.java

```diff
@@ -71,6 +75,9 @@
 
 	private static final String PROPERTY_PREFIX = "spring.cloud.appbroker";
 
+	@Value("${spring.application.name:unknown}")
```

Comment:
Shouldn't we fail-fast instead of providing a default value?

Reply:
@AlbertoImpl I'm fine with that. I'll remove the default. Thanks.